### DISCUSSION TITLE
Found it... data capture - still scary?

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#40] [BRAT] [answered]
Hey @philipbaileynar. I found the [data capture post in pyBRAT](github.com/Riverscapes/pyBRAT/issues/31). It is fairly specific to capturing dam observations.  Not sure, but thought I'd ask you to have a look and see what you think.

**Comment by philipbaileynar** [comment, marked as answer]
Nothing scary. Ultimately I think a dynamo database for riverscapes point observations of all types. I suggest dynamo because it stores objects as simple JSON data objects. Each record can have different attributes. So storing beaver activity and vegetation validation in the same database is possible. Also, it allows for attributes to change over time as the protocol evolves. This central store could be used for several models, tools and purposes. BRAT, citizen beaver etc.
